dict.de.css

```diff
@@ -89,3 +89,11 @@ a.action.load-more::before {
 .filter-wrapper .filter:first-child::before {
   content: "Filter";
 }
+
+.filter .products button::before {
+  content: "Produkten";
```

Comment:
```suggestion
  content: "Produkte";
```

Reply:
Just updated that @rofe and about 
And maybe you could add localized titles to the dropdowns in i18n(), similar to how we do it for the buttons?
will create an issue labeled as enhancement for this one.